bundles/pixi-modern/README.md

```diff
@@ -0,0 +1,14 @@
+PixiJS Modern
```

Comment:
Not sure as of `modern` name here. I would think of it as something "new and unrelated to old pixi" .

Do you think there is no better options?

Reply:
I agree with @cursedcoder, although in true critic fashion I can't think of anything better! How about pixi-webgl or something like that?

Reply:
@GoodBoyDigital `pixi-canary`?

Reply:
I chatted offline with @GoodBoyDigital about this, we we discussed that **pixi.js** should be the package _without_ canvas support. Much of what we're doing is making WebGL first-class and Canvas 2nd-class for v5. This change should also reflect that. The current **pixi.js** in next would be renamed to  **pixi.js-legacy** (or something similar) to have fallback-support for CanvasRenderer and compatibility with older browsers.

Reply:
I had this in my mind but didn't think you want to drop canvas that hard.

Big +1 :D